pandas/tests/arrays/masked/test_arithmetic.py

```diff
@@ -52,6 +54,11 @@ def test_array_scalar_like_equivalence(data, all_arithmetic_operators):
 def test_array_NA(data, all_arithmetic_operators):
     if "truediv" in all_arithmetic_operators:
         pytest.skip("division with pd.NA raises")
+    if (
+        "floordiv" in all_arithmetic_operators
```

Comment:
all_arithmetic_operators is a scalar, despite the name. maybe should be `==` and add the dunders for clarity?

and then maybe simplify further...

```
if all_arithmetic_operators in ("__floordiv__", "__rfloordiv__"):
```

Reply:
See below